documentation/sile.sil

```diff
@@ -952,19 +952,35 @@ for instance, you may wish to write a thesis like this:
 \begin{verbatim}
 \line
 \\begin[class=thesis]\{document\}
-\\include[src=macros]
-\\include[src=chap1]
-\\include[src=chap2]
-\\include[src=chap3]
+\\include[src=macros.xml]
+\\include[src=chap1.sil]
+\\include[src=chap2.sil]
+\\include[src=chap3.sil]
 \dots\par
-\\include[src=endmatter]
+\\include[src=endmatter.sil]
 \\end\{document\}
 \line
 \end{verbatim}
 
 \code{\\include}s may be nested, in that file A can include file B which includes
 file C.
 
+The contents of an included file should be put in a \code{\\sile} tag (or
+\code{<sile>} if the file is in XML flavour), like so:
+
+\begin{verbatim}
+\line
+\\begin\{sile\}
+  \\chapter\{A Scandal In Bohemia\}
+
+  To Sherlock Holmes she is always \em\{the woman\}.
```

Comment:
```suggestion
\\chapter\{A Scandal In Bohemia\}

To Sherlock Holmes she is always \em\{the woman\}.
```

Unlike in most programming languages where white-space is discarded and can be used to visually indent something in a block, leading white-space in the TeX-Like markup here can actually have an effect on the output, usually undesired. I would recommend not indenting lines like this.